src/lib/Controls.js

```diff
@@ -24,6 +25,18 @@ class Controls {
     /** @property {boolean} - Whether browser supports touch */
     hasTouch = Browser.hasTouch();
 
+    /** @property {HTMLElement} - Page num input element */
+    pageNumInputEl;
+
+    /** @property {String} - HTML template for page num element */
+    pageNumTemplate = `
```

Comment:
Instead of making this a property of Controls.js (since it is not required for base controls, and is used as a constant) could you make this an exported `constant`